rules.d/60-dmi-id.rules

```diff
@@ -14,6 +14,10 @@ ENV{ID_MODEL}=="", ENV{ID_MODEL}="$attr{board_name}"
 # stock keeping unit
 ENV{ID_PRODUCT_SKU_IS_RUBBISH}!="1", ENV{ID_SKU}="$attr{product_sku}"
 
+# hardware version
+ENV{ID_PRODUCT_VERSION_IS_RUBBISH}!="1", ENV{ID_HWVERSION}="$attr{product_version}"
+ENV{ID_HWVERSION}=="", ENV{ID_BOARD_VERSION_IS_RUBBISH}!="1", ENV{ID_HWVERSION}="$attr{board_version}"
```

Comment:
I'd like to make the property not abbreviated: ID_HWVERSION -> ID_HARDWARE_VERSION

Reply:
I changed it.